python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -237,6 +237,8 @@ def _getDefaultValueInternal(self, attr, val=None, check=False):
         from Ganga.Utility.Config import Config
         is_finalized = Config._after_bootstrap
 
+        _useDefVal = False
```

Comment:
This doesn't need to be prefaced with an `_` as it's not being exposed to any external interface.